src/inmanta/loader.py

```diff
@@ -208,7 +211,7 @@ def _load_module(self, mod_name: str, hv: str) -> None:
                 mod = importlib.import_module(mod_name)
             self.__modules[mod_name] = (hv, mod)
             LOGGER.info("Loaded module %s" % mod_name)
-        except ImportError:
+        except (ImportError, PluginModuleLoadException):
```

Comment:
I think this is right but I'm not entirely sure. This will cause more exceptions to be caught and logged than before.

Reply:
I think this is correct.